templates/zerver/emails/digest.html

```diff
@@ -1,32 +1,49 @@
 {% extends "zerver/emails/email_base_messages.html" %}
 
 {% block content %}
-    {% if hot_conversations %}
-        {% for convo in hot_conversations %}
-        <div class='messages'>
-            {% for recipient_block in convo.first_few_messages %}
-                <div class='hot_convo_recipient_block'>
-                    <div class='hot_convo_recipient_header'>{{ recipient_block.header.html|safe }}</div>
-                    <div class='hot_convo_message_content'>
-                        {% for sender_block in recipient_block.senders %}
-                            {% for message_block in sender_block.content %}
-                            <div class='hot_convo_message_content_block'>
-                                {{ message_block.html|safe }}
-                            </div>
+    {% if message_content_disabled_by_realm %}
+        {% if new_messages_count > 0 or new_streams_count > 0 %}
+            <p class="digest_paragraph">
+                {% if new_messages_count > 0 and new_streams_count > 0 %}
+                    {% trans %}You have {{ new_messages_count }} new messages and {{ new_streams_count }} new channels in <a href="{{ realm_url }}">{{ realm_name }}</a>.{% endtrans %}
+                {% elif new_messages_count > 0 and new_streams_count == 0 %}
+                    {% trans %}You have {{ new_messages_count }} new messages in <a href="{{ realm_url }}">{{ realm_name }}</a>.{% endtrans %}
+                {% else  %}
+                    {% trans %}You have {{ new_streams_count }} new channels in <a href="{{ realm_url }}">{{ realm_name }}</a>.{% endtrans %}
+                {% endif %}
+            </p>
+            <p class="digest_paragraph">
+                {% trans help_url=realm_url + "/help/hide-message-content-in-emails" %}This email does not include message content because your organization <a class="content_disabled_help_link" href="{{ help_url }}">hides message content</a> in email notifications.{% endtrans %}
```

Comment:
Hmm... the way this code is currently, it could be either the realm or user setting that's causing the message content to be hidden. So maybe we  want to pass both of those as separate parameters, with two distinct messages, like we do in the missed message emails.